Ports/README.md

```diff
@@ -11,6 +11,16 @@ environment.
 
 A list of all available ports can be found [here](AvailablePorts.md).
 
+## External ports
+
+Port directories outside of the Serenity repository can be added by setting the `SERENITY_PORT_DIRS` environment variable.
```

Comment:
I think this can be improved by explaining that you need to point to the root of the ports directory. I.e.

````
You can add third party ports by pointing to a local ports directory in the `SERENITY_PORT_DIRS` variable. For example:

```bash
export SERENITY_PORT_DIRS=("/path/to/Ports/")
```
````

Reply:
Also, I don't think this is a true "environment" variable, since those can only be strings I believe? This is more a local/shell variable defined as an array.

Reply:
True, arrays are a bash concept that can only be defined in the current process, and this made me realize an issue with this that I had previously fixed, but wrongly removed after forgetting what its intention was.

`SERENITY_PORT_DIRS` does not get passed over when .port_include.sh is invoked, since the variable is not set as an environment variable.

The solution to this would be to source .port_include.sh, which preserves the local environment, and requires only a change to the initial `SCRIPT` declaration.

Reply:
Also: when `SERENITY_PORT_DIRS` is defined as a simple string, append to it as if it were a list converts it to a list.

Since invoking `package.sh` essentially invokes `bash ../port-include.sh ./package.sh` this means setting `SERENITY_PORT_DIRS` as an array before invoking it will result in the variable never being passed, meaning it will always require to be included.

Reply:
Moved over to a string based environment variable that is colon seperated.